### ISSUE TITLE
Consider using OpenTheory for persistent theories

### ISSUE THREAD

**Issue by xrchz** [HOL-Theorem-Prover/HOL#82]
See discussion on #71. Could also be useful alongside DISK_THM-style persistent theories.
##

**Comment by xrchz** [comment]
Is this done given that we now have otknl?

**Comment by mn200** [comment]
I don't think so.  I assume the original suggestion was to store open-theory proofs as permanent records of theorems.  In contrast, the `otknl` approach *generates* open-theory proofs as build products.
